src/index.js

```diff
@@ -75,6 +75,56 @@ var _applyToPath = function(obj, path, val) {
   }
 };
 
+var doConcat = function(obj, args) {
```

Comment:
Wait, how does lodash help here? What would the lodash implementation look like?